conda/environments/all_cuda-129_arch-aarch64.yaml

```diff
@@ -16,21 +15,21 @@ dependencies:
 - cpp-argparse
 - cuda-nvcc
 - cuda-nvtx-dev
-- cuda-python>=12.9.2,<13.0a0
+- cuda-python>=12.9.2,<13.0
 - cuda-sanitizer-api
 - cuda-version=12.9
 - cudf==26.2.*,>=0.0.0a0
 - cupy>=13.6.0
 - cxx-compiler
 - cython>=3.0.3
 - doxygen=1.9.1
-- exhale
```

Comment:
The `exhale` dependency pins `sphinx` tightly (`<5`) and we require newer Sphinx versions in most of RAPIDS. This doesn't appear to be used so I removed it.